wagn/lib/wagn/generators/wagn/templates/config/databases/mysql.yml

```diff
@@ -1,11 +1,12 @@
 # MySQL.  Versions 4.1 and 5.0 are recommended.
 # Be sure to use new-style password hashing:
 #   http://dev.mysql.com/doc/refman/5.0/en/old-client.html
-development:
+
+production: 
   adapter: mysql2
   encoding: utf8
   reconnect: false
-  database: <%= app_name %>_development
+  database: <%= app_name %>_production
   pool: 5
   username: root
   password:
```

Comment:
moved production to the top, because it's the main db for folks creating new wagn apps (unlike those creating new rails apps).  Would like to simplify this further...